src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -375,6 +375,46 @@ class AdminApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  it should "return 200 when listing users" in withTestDataApiServices { services =>
```

Comment:
Add a new no-testData user, in case the already-loaded testData is special in some way